cache/file/file.go

```diff
@@ -0,0 +1,64 @@
+// Package file provides a disk based cache.
+package file
+
+import (
+	"fmt"
+	"github.com/pierrre/imageserver"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+)
+
+// Cache is a implementation of disk based cache system.
+type Cache struct {
+	Path  string
+	Mutex sync.RWMutex
```

Comment:
As I said previously,  we should name this field `mu sync.RWMutex`.
We don't need to expose it (it's just a private field).